libevmasm/Assembly.h

```diff
@@ -235,6 +235,28 @@ class Assembly
 	/// Returns map from m_subs to an index of subcontainer in the final EOF bytecode
 	std::map<uint16_t, uint16_t> findReferencedContainers() const;
 
+	/// Assembly items types handlers to be used by the dispatching functions for EOF or legacy.
+	void handleOperation(AssemblyItem const& item, bytes& bytecode) const;
+	void handlePush(AssemblyItem const& item, bytes& bytecode) const;
+	void handlePushLibraryAddress(AssemblyItem const& item, LinkerObject& ret) const;
+	void handleVerbatimBytecode(AssemblyItem const& item, bytes& bytecode) const;
+	void handlePushDeployTimeAddress(bytes& bytecode) const;
+	void handleTag(AssemblyItem const& item, bytes& bytecode, bool addJumpDest) const;
+
+	/// Iterates and dispatches assembly items and returns needed references types for further bytecode construction.
+	/// Returns tagRef, dataRef, subRef, sizeRef
+	std::tuple<std::map<size_t, std::pair<size_t, size_t>>, std::multimap<util::h256, unsigned>, std::multimap<size_t, size_t>, std::vector<unsigned>>
+	dispatchAssemblyItemsLegacy(
+		LinkerObject& ret,
+		std::map<u256, std::pair<std::string, std::vector<size_t>>>& immutableReferencesBySub,
+		AssemblyItems const& items,
+		unsigned bytesPerTag,
+		unsigned bytesPerDataRef) const;
```

Comment:
```suggestion
		unsigned bytesPerDataRef
	) const;
```

Same in the `.cpp` file.